src/Tasks/ManifestUtil/Util.cs

```diff
@@ -29,11 +29,12 @@ internal static class Util
         internal static readonly string logPath = GetLogPath();
         private static readonly char[] s_fileNameInvalidChars = { '\\', '/', ':', '*', '?', '"', '<', '>', '|' };
         private static StreamWriter s_logFileWriter;
-        // Major, Minor, Build and Revision of CLR v2.0
-        private static readonly int[] s_clrVersion2 = { 2, 0, 50727, 0 };
 #if RUNTIME_TYPE_NETCORE
         // Major, Minor, Build and Revision of CLR v4.0
         private static readonly int[] s_clrVersion4 = { 4, 0, 30319, 0 };
+#else
```

Comment:
nit:
Maybe 
private static readonly int[] s_clrVersion =
#if NETFRAMEWORK
{ 2, 0, 50727, 0 };
#else
{ 4, 0, 30319, 0 };
#endif

?
Having the older version first makes more sense to me; I'm less sure about combining them because it makes it cleaner but also prevents us from using quite as clear names.

Reply:
Sorted. Merging might make sense at some point. but I think it should probably be apart of a refactor of the whole class.

Reply:
Sounds good